tests/ui/parser/no-const-fn-in-extern-block.stderr

```diff
@@ -7,22 +7,19 @@ LL |     const fn foo();
    |     ^^^^^ help: remove this qualifier
 
 error: functions in `extern` blocks cannot have qualifiers
-  --> $DIR/no-const-fn-in-extern-block.rs:4:11
+  --> $DIR/no-const-fn-in-extern-block.rs:4:5
    |
 LL | extern "C" {
    | ---------- in this `extern` block
 ...
 LL |     const unsafe fn bar();
-   |           ^^^^^^ help: remove this qualifier
+   |     ^^^^^ help: remove this qualifier
 
-error: functions in `extern` blocks cannot have qualifiers
+error: safety is not expected in unadorned extern blocks
   --> $DIR/no-const-fn-in-extern-block.rs:4:5
    |
-LL | extern "C" {
-   | ---------- in this `extern` block
```

Comment:
this marker was lost

Reply:
OMG, hehehe, I was tricked the same way you were tricked by this :). Was investigating this issue until I figured it was not possible, then I've checked this diff again and is not lost :). Look at line 16 in the right :).
What is probably wrong is the thing we have in line 22, which instead of highlighting unsafe we highlight everything. Which I guess can potentially be fixed by moving this check to ast validation as you've already mentioned.

Reply:
🤦